contracts/scripts/deploy.ts

```diff
@@ -51,10 +51,13 @@ async function main() {
     deploymentsStringified
   );
 
-  //Copy the ABI from `artifacts/src/Summa.sol/Summa.json` to `backend/src/contracts/Summa.json`
-  const abi = require("../artifacts/src/Summa.sol/Summa.json");
-  const abiStringified = JSON.stringify(abi);
-  fs.writeFileSync("../backend/src/contracts/Summa.json", abiStringified);
+  //Copy the ABIs from `artifacts/src/*` to `backend/src/contracts/*`
```

Comment:
Can you add a reference to what you are performing with this script inside the `readme.md`